charts/rstudio-pm/templates/NOTES.txt

```diff
@@ -11,6 +11,17 @@ NOTE: `.Values.config.Licensing.LicenseType` is configured manually. Normally, w
 Please consider removing this configuration value.
 {{- end }}
 {{- end }}
+
+{{- if hasKey .Values.config "Launcher" }}
+  {{- if hasKey .Values.config.Launcher "ServerUser" }}
+    {{- if eq (get .Values.config.Launcher "ServerUser") "root" }}
+      {{- if .Values.rootCheckIsFatal }}
+        {{- fail "\n\n`config.Launcher.ServerUser=root`, but the public helm chart and docker image now run Package Manager as non-root. Please remove the `root` values from your values file.\n\nDANGER: set `.Values.rootCheckIsFatal` to bypass this error." }}
```

Comment:
I'm not sure we need to fail here, I think a warning is fine. Package Manager itself has been updated to log a useful error message in this case that should explain the remedy.

Reply:
I am inclined to just remove this whole block - it's easier that way for sure. The one reason I have pause is because anyone who copy / pasted the values file (I have seen this in practice a good bit) to set their defaults _will_ hit this issue, so the faster turnaround cycle seems preferable. i.e. we set them up for this problem w/ not-great values before.

I'm not sure why this is a convention that folks adhere to... but it makes me wonder whether we should keep the extra check or not.